README.md

```diff
@@ -490,7 +489,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Ayushi Vamne](https://github.com/Ayushi15-Vamne) 
   - [Aziz Prabowo](https://github.com/azizp128)
   
-| [Back To Top](#contents) |
+| [`Back To Top`](#contents) |
```

Comment:
Hmm, I think we should remove the code backticks, as well as in all the other instances of this.